zhmcclient/_nic.py

```diff
@@ -0,0 +1,183 @@
+# Copyright 2016 IBM Corp. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+A **NIC** object represents a single Network interface card (NIC)
+of a physical z Systems or LinuxONE computer that is in DPM mode
+(Dynamic Partition Manager mode).
```

Comment:
This sentencer suggests that the term NIC is used for the physical network adapter. The last sentence of this paragraph describes that better, and could be used as the definition of what a NIC is.